internal/tracker/source/tekton/tekton.go

```diff
@@ -553,5 +554,20 @@ func enrichPackageFromAnnotations(p *hub.Package, annotations map[string]string)
 		}
 	}
 
+	// Sign key
+	if v, ok := annotations[signatureAnnotation]; ok {
+		var signature string
+		if err := yaml.Unmarshal([]byte(v), &signature); err != nil {
+			errs = multierror.Append(errs, fmt.Errorf("%w: invalid signature value", errInvalidAnnotation))
+		} else {
+			if signature == "" {
+				errs = multierror.Append(errs, fmt.Errorf("%w: signature value not provided", errInvalidAnnotation))
+			} else {
+				p.Signatures = []string{signature}
```

Comment:
We use the `signatures` field to track the kind of signatures found in the package, not the signature itself (please see signature in [containers images](https://github.com/artifacthub/hub/blob/b0a38faed699952c3202001693e5b933aac8a078/internal/tracker/source/container/container.go#L329) or in [Helm charts](https://github.com/artifacthub/hub/blob/b0a38faed699952c3202001693e5b933aac8a078/internal/tracker/source/helm/helm.go#L288-L315) as an example). Given this is a new kind of signature specific for Tekton, we could define a constant for it with the `tekton` value.

Reply:
Sorry for missing the point! Made the change as you suggested.